src/pybind/mgr/dashboard/tox.ini

```diff
@@ -139,3 +141,13 @@ commands =
 [testenv:check]
 commands =
     python ci/check_grafana_uids.py frontend/src/app ../../../../monitoring/grafana/dashboards
+
+[testenv:apidocs]
+deps =
+    {[base]deps}
+    {[base-test]deps}
+setenv =
+    UNITTEST = true
+changedir = ..
+commands =
+    python3 -m dashboard.controllers.docs {posargs}
```

Comment:
* You still need to change this, to run from the mgr/dashboard/ dir instead of from the parent one.
* Rebase the whole branch, as there's a conflict.
* Change the commit message format: 
  * title: 'mgr/dashboard: generate offline etc.'
  * body: you have to specify the `Fixes: https://tracker.ceph.com/...` issue you are fixing with this.

Reply:
PR has been rebased, and fixed, and Fixes: added